native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -39,24 +46,83 @@
 import static io.ballerina.persist.PersistToolsConstants.DEFAULT_USER;
 
 /**
- * Class to create syntax tree for ballerina.
+ * Class to create syntax tree for config.toml.
  */
 public class CreateSyntaxTree {
     private static final PrintStream outStream = System.out;
+
+    /**
+     * Method to create new config.toml.
+     */
     public static SyntaxTree createToml() {
+        NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        moduleMembers = populateNodeList(moduleMembers, true);
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
+        return syntaxTree;
+    }
 
+    /**
+     * Method to update config.toml.
+     */
+    public static SyntaxTree updateToml(Path configPath) throws IOException {
+        boolean withinTable = false;
         NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        moduleMembers = populateNodeList(moduleMembers, false);
+        Path fileNamePath = configPath.getFileName();
+        TextDocument configDocument = TextDocuments.from(Files.readString(configPath));
+        SyntaxTree syntaxTree = SyntaxTree.from(configDocument, fileNamePath.toString());
+        DocumentNode rootNote = (DocumentNode) syntaxTree.rootNode();
+        NodeList nodeList = rootNote.members();
+
+        for (Object i : nodeList) {
```

Comment:
Let's change the variable `i` -> `member`

Reply:
updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)